packages/withoutHTML/test/index.js

```diff
@@ -0,0 +1,13 @@
+import withoutHTML from '../src';
+
+it('removes <b> tags from HTML', () => {
+  expect(withoutHTML('<b>Testing</b>')).toBe('Testing');
+});
+
+it('removes multiple tags from HTML', () => {
+  expect(withoutHTML('<b><i>Testing</i></b>')).toBe('Testing');
+});
+
+it('removes tags from a HTML table', () => {
```

Comment:
Can you please add a few more test cases with HTML attributes?

Reply:
Added, but will add more once I have enough time.